src/Utilities/System/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ spectre_target_sources(
   ${LIBRARY}
   PRIVATE
   Abort.cpp
+  Exit.cpp
```

Comment:
Did you intend to actually make the dependence on Charm `PRIVATE` in this file as the commit message promises? If so I think it would be good to also add a comment there why you made it private so it won't accidentally be made public again later.

Reply:
I did, but I couldn't get it to work because we still have some cyclic library dependencies.  I'll remove the comment from the commit message

Reply:
Do you understand the cyclic dependency? How can making a dependency private introduce a cyclic dependency?